diff --git a/node_modules/@particle-network/wallet/dist/esm/index.mjs b/node_modules/@particle-network/wallet/dist/esm/index.mjs
index 4be3365..ae8cb8f 100644
--- a/node_modules/@particle-network/wallet/dist/esm/index.mjs
+++ b/node_modules/@particle-network/wallet/dist/esm/index.mjs
@@ -513,7 +513,12 @@ function walletUrl({
         return {
           name: chain.name,
           id: chain.id,
-          chainType: chain.chainType
+          chainType: chain.chainType,
+          rpcUrl: chain.rpcUrl,
+          icon: chain.icon,
+          blockExplorerUrl: chain.blockExplorerUrl,
+          testnet: chain.testnet,
+          nativeCurrency: chain.nativeCurrency
         };
       });
     }
