diff --git a/node_modules/@particle-network/wallet-plugin/dist/esm/index.mjs b/node_modules/@particle-network/wallet-plugin/dist/esm/index.mjs
index 9ec9a31..5613cbf 100644
--- a/node_modules/@particle-network/wallet-plugin/dist/esm/index.mjs
+++ b/node_modules/@particle-network/wallet-plugin/dist/esm/index.mjs
@@ -120,6 +120,7 @@ var wallet = (parameters = {}) => {
             ...(_f = this.getPlugin().walletOptions) == null ? void 0 : _f.customStyle,
             supportChains: validChains.map((chain) => ({
               id: chain.id,
+              name: chain.name,
               chainType: connector.chainType
             }))
           }
@@ -136,6 +137,7 @@ var wallet = (parameters = {}) => {
               ...(_h = (_g = this.getPlugin()) == null ? void 0 : _g.walletOptions) == null ? void 0 : _h.customStyle,
               supportChains: validChains.map((chain) => ({
                 id: chain.id,
+                name: chain.name,
                 chainType: connector.chainType
               }))
             }
